tests/conftest.py

```diff
@@ -97,6 +97,11 @@ def pytest_addoption(parser):
     parser.addoption("--logs_since", action="store", type=int,
                     help="number of minutes for show techsupport command")
 
+    # fw_utility options
+    parser.addoption("--config_file", action="store", default=None, help="name of configuration file (per each vendor)")
+    parser.addoption("--binaries_path", action="store", default=None, help="path to binaries files")
+    parser.addoption("--second_image_path", action="store", default=None, help="path to second image to be installed if there is no image available")
```

Comment:
What are these changes? They don't seem to be related to the fanout fixture

Reply:
Fixed